modules/dxp/apps/search-experiences/search-experiences-web/src/main/java/com/liferay/search/experiences/web/internal/blueprint/admin/display/context/ViewSXPBlueprintsDisplayContext.java

```diff
@@ -97,21 +97,17 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 				LanguageUtil.get(_sxpRequestHelper.getRequest(), "edit"), "get",
 				"get", null),
 			new FDSActionDropdownItem(
-				getAPIURL() +
-					"/{id}/set-as-collection-provider?collectionProvider=true",
-				"book", "enableAsACollectionProvider",
+				"#", "book", "enableAsACollectionProvider",
 				LanguageUtil.get(
 					_sxpRequestHelper.getRequest(),
 					"enable-as-a-collection-provider"),
-				"post", "enable", "async"),
+				"put", "update", null),
```

Comment:
`target` cannot be null otherwise it'll have issues with the data set manager if it ever got customized through it. I'd suggest trying `"link"` for this since it's action is enhanced with a propTransformer.

Reply:
Oh I didn't know that, thanks for letting me know! I'll use "link" and test it out, just to make sure it works the same.

Reply:
Ya I recently discovered that when updating our tests for the sample FDS widget.